### PR TITLE
Acknowledge timeout

### PR DESCRIPTION
**Pull request by miiu96**
- Return error on `Send` method in case of an acknowledgment timeout.

### CODE REVIEW

File: websocket/data/errors.go [hunk 1]
```diff
@@ -34,3 +37,6 @@ var ErrInvalidWebSocketHostMode = errors.New("invalid web socket host mode")
 
 // ErrNoClientsConnected is an error signal indicating the absence of any connected clients
 var ErrNoClientsConnected = errors.New("no client connected")
+
+// ErrAckTimeout signal that acknowledge timeout has been reached
+var ErrAckTimeout = errors.New("wait acknowledge timeout occurred")
```
Comment (bogdan-rosianu) [inline websocket/data/errors.go:42]: errors.New("acknowledge waiting timeout occurred")

File: websocket/data/errors.go [hunk 2]
```diff
@@ -34,3 +37,6 @@ var ErrInvalidWebSocketHostMode = errors.New("invalid web socket host mode")
 
 // ErrNoClientsConnected is an error signal indicating the absence of any connected clients
 var ErrNoClientsConnected = errors.New("no client connected")
+
+// ErrAckTimeout signal that acknowledge timeout has been reached
```
Comment (bogdan-rosianu) [inline websocket/data/errors.go:41]: signals that an

File: websocket/data/errors.go (hunk 1 above)
Comment (miiu96) [reply websocket/data/errors.go:42]: done

File: websocket/data/errors.go (hunk 2 above)
Comment (miiu96) [reply websocket/data/errors.go]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Review by iulianpascalau — DISMISSED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**